src/website/src/assets/images/documentation/timeline/all-step-descriptions.svg

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Nit: won't block the PR on this but it would be nice to run the SVGs through an optimizer like https://jakearchibald.github.io/svgomg/ to rove the unnecessary artifacts from Sketch.

Reply:
Great suggestion, I ran it through the tool and pushed the update. 

offhand but kind of related, do you know if the cli webpack is configured to optimize assets when it builds for production?

Reply:
It doesnt touch anything in the assets directory. My guess is they don't want to slow down the build any more than they have to especially on files that rarely change.